_includes/enterprise_sidebar.html

```diff
@@ -33,37 +33,32 @@ <h3>Travis CI Enterprise</h3>
         <li><a href="/user/enterprise/">About Travis CI Enterprise</a></li>
         <li><a href="/user/enterprise/setting-up-travis-ci-enterprise/">Setting up Travis CI Enterprise</a> </li>
         <li><a href="/user/enterprise/upgrading/">Upgrading</a></li>
-
-
       </ul>
 
-        <h3>Environment reference</h3>
-        <ul>
-          <li><a href="/user/enterprise/xenial">Xenial Build Environment</a></li>
+      <h3>Environment reference</h3>
+      <ul>
+        <li><a href="/user/enterprise/xenial">Xenial Build Environment</a></li>
 
-          <li><a href="/user/enterprise/trusty/">Trusty Build Environment</a></li>
+        <li><a href="/user/enterprise/trusty/">Trusty Build Environment</a></li>
 
-          <li><a href="/user/enterprise/precise/">Precise (Legacy) Build Environment</a></li>
-        </ul>
+        <li><a href="/user/enterprise/precise/">Precise (Legacy) Build Environment</a></li>
+      </ul>
 
-        <h3>Customizing Travis CI Enterprise</h3>
+      <h3>Customizing Travis CI Enterprise</h3>
       <ul>
-
         <li><a href="/user/enterprise/worker-configuration/">Worker Configuration</a></li>
         <li><a href="/user/enterprise/build-images/">Customizing Build Images</a></li>
         <li><a href="/user/enterprise/high-availability/">High Availability</a></li>
         <li><a href="/user/enterprise/custom-queues/">Custom Queues</a></li>
-
       </ul>
 
       <h3>Enterprise Operations Manual</h3>
       <ul>
         <li><a href="/user/enterprise/platform-tips/">Platform Administration Tips</a></li>
         <li><a href="/user/enterprise/worker-cli-commands/">Worker Start & Debug Containers</a></li>
-        <li><a href="/user/enterprise/operations-manual/">Operations Manual</a></li>
+        <li><a href="/user/enterprise/troubleshooting-guide/">Troubleshooting Guide</a></li>
```

Comment:
This is the only real change in this file, the rest is just indentation updates.

Reply:
:+1: I think @schultyy has thoughts about the naming of Ops Manual, but can't I remember what they are.

Reply:
The Operations Manual was meant as a template for customers so they can use that document as a basis for their own troubleshooting runbooks. I'd like to keep the guide itself, but I'm not particularly tied to the name.

Reply:
Great, sounds like we are in agreement, then! The actual content of the document hasn't changed outside of language/structure so we should be good to go. 👍